packages/android_alarm_manager_plus/test/android_alarm_manager_test.dart

```diff
@@ -12,6 +11,10 @@ void main() {
   String invalidCallback(String foo) => foo;
   // ignore: avoid_returning_null_for_void
   void validCallback(int id) => null;
+  // ignore: avoid_returning_null_for_void
+  void invalidCallbackWithParams(int id, List<String> params) => null;
+  // ignore: avoid_returning_null_for_void
+  void validCallbackWithParams(int id, Map<String, dynamic> params) {}
```

Comment:
Why does this have a lint ignore and doesn't return null, like the methods above?